mailbox/api/src/main/java/org/apache/james/mailbox/MessageIdManager.java

```diff
@@ -32,7 +32,7 @@
 
 public interface MessageIdManager {
 
-    void setFlags(Flags newState, FlagsUpdateMode replace, MessageId messageId, MailboxId mailboxId, MailboxSession mailboxSession) throws MailboxException;
+    void setFlags(Flags newState, FlagsUpdateMode replace, MessageId messageId, MailboxSession mailboxSession) throws MailboxException;
```

Comment:
Ok. I will adapt.

@aduprat  actually removed the Mailbox field from his API (hence this commit). I will re-add it. And notify @aduprat